metriq/models/__init__.py

```diff
@@ -40,3 +41,9 @@
     MethodCreateRequest,
     MethodUpdateRequest,
 )
+from metriq.models.moderationReport import (
+    ModerationReport,
+    ModerationReports,
+    ModerationReportCreateRequest,
+    ModerationReportUpdateRequest,
```

Comment:
`ModerationReports` (plural) and `ModerationReportUpdateRequest` aren't needed. Please remove them.